src/game/camera.c

```diff
@@ -1148,13 +1239,18 @@ void mode_8_directions_camera(struct Camera *c) {
         s8DirModeYawOffset = snap_to_45_degrees(s8DirModeYawOffset);
     }
 #endif
-
+#endif
```

Comment:
```suggestion
#endif // REONUCAM
```

Reply:
and `// PARALLEL_LAKITU_CAM` on the line above cus its a bit confusing whats starting and whats ending. but also you could just saveguard parallel lakitu cam from being enabled